keps/sig-scheduling/624-scheduling-framework/README.md

```diff
@@ -603,9 +603,7 @@ kubescheduler.config.k8s.io/v1alpha1`. This new config format will be either
 configuration.
 
 *Note: Moving `KubeSchedulerConfiguration` to `v1` is outside the scope of this
-design, but see also
-https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/wgs/0032-create-a-k8s-io-component-repo.md
-and https://github.com/kubernetes/community/pull/3008*
+design, but see also https://github.com/kubernetes/community/pull/3008*
```

Comment:
I think the broken link was moved to https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/wgs/783-component-base/README.md, so could you update the link instead of removing it?

Reply:
@Huang-Wei I Will close this pull request and create a new one with the required changes